### ISSUE TITLE
Nothing is working after installing

### ISSUE THREAD

**Issue by Jensen0710** [atom/ide-csharp#11]
I can't seem to get this package to work. I don't get the errors i should, and the outline says "No outline available, this is likely an error with the language package ". 
The packages for typescript and php is working.
<img width="308" alt="screen shot 2017-10-27 at 21 48 52" src="https://user-images.githubusercontent.com/9888000/32133600-656fd160-bbdb-11e7-8b1c-565badaf47b0.png">

**Comment by rsese** [comment]
Can you confirm what version of Atom you're running (`atom -v`) and what operating system?  Also, can you open dev tools console and run this command:

```
atom.config.set('core.debugLSP', true)
```

And then restart Atom and share the output after trying the outline view?

Also, are you seeing an empty hover as mentioned in https://github.com/atom/ide-csharp/issues/9?

**Comment by daviwil** [comment]
@Jensen0710 Are you trying to use this package on Linux?  This may be related to issue #7.

**Comment by JosephTLyons** [comment]
I'm having the exact same issues as OP.  Nothing is different after installing the atom UIs and this package.  Same error message in the outline view, zero errors, zero warnings, no difference in autocompletion.

I'm on 1.22.0 and on macOS 10.12.6

**Comment by dimixar** [comment]
[-1512463147032.log](https://github.com/atom/ide-csharp/files/1530289/-1512463147032.log)
Here's the log. I have the same problem as these guys.
And this:

atom -v

Atom    : 1.22.1
Electron: 1.6.15
Chrome  : 56.0.2924.87
Node    : 7.4.0

**Comment by dimixar** [comment]
Any new info about this bug.

I noticed that sometimes it starts to work. From logs, it seemed like it gives completion suggestions very late, and it's not even sorted by the parts of the text I write.

**Comment by JosephTLyons** [comment]
Really wish there would be some kind of update on this.  It just appears as if there is no active development on this.

**Comment by rsese** [comment]
@dimixar are you working on a particularly large project or do you see the same behavior on a small test project?

@JosephTLyons can you confirm if nothing works for you on any project as well?  If so can you share a simple test project that where you're not seeing any functionality so I can test on my Mac?  Our engineers need as much information as possible to reproduce a problem before they can investigate so any details you can share would be helpful.

**Comment by dimixar** [comment]
I tried on a smaller project too, had the same effect. I can create a test project and test it again with a single script.

**Comment by JosephTLyons** [comment]
The only C# projects I work with are Unity projects.  I don't work with any outside of that.  If anyone is willing to send me some small test file, I'd gladly run it and test it.

**Comment by dimixar** [comment]
I'm also using Unity mostly. Tell me what can I send you to make it easier.

**Comment by damieng** [comment]
We've been put a bit on hold as the underlying wrapper we used to connect to Omnisharp was put aside in favor of building direct Language Server Support into Omnisharp-Roslyn directly.

I've reached out to the Omnisharp team for their thoughts on switching over, the current state of LSP support and how best to distribute their language server with our package.

Stay tuned!

**Comment by JosephTLyons** [comment]
Thank you for the update, I was beginning to think that this C# support for Atom IDE was going to become abandonware.  Thanks.

**Comment by JosephTLyons** [comment]
Here is a simple hello world program I got from [Microsoft's page](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/hello-world-your-first-program)

You can see that even though I inserted random text in the doc, there is no error or warnings, and outline mode is still displaying nothing.

![screen shot 2017-12-14 at 3 45 04 pm](https://user-images.githubusercontent.com/19867440/34013536-d8823ca4-e0e5-11e7-9097-03ccd573306a.png)

**Comment by rsese** [comment]
@JosephTLyons I think that's related to my comment here /cc https://github.com/atom/ide-csharp/issues/4#issuecomment-347482820.  I found that you do have to have a .csproj file for your project - once you add that and reload the window, some of the functionality should start working.

**Comment by JosephTLyons** [comment]
@rsese I'm not sure if I did this correctly, but if I did, its still not working on my end:

![screen shot 2017-12-15 at 12 04 57 am](https://user-images.githubusercontent.com/19867440/34027592-a9d4a5ea-e12b-11e7-936c-f535411cdba8.png)

**Comment by JosephTLyons** [comment]
Here's a unity project as well:

![screen shot 2017-12-15 at 12 09 52 am](https://user-images.githubusercontent.com/19867440/34027692-5db9f07e-e12c-11e7-8274-372802da25c1.png)

**Comment by rsese** [comment]
@JosephTLyons - sorry for the hello world program go ahead and give the file a name in addition to the extension e.g. `Hello.csproj` and then start Atom from that directory with `atom .`.   That's what I did to get past the point where it seems like nothing was working.  A lot of the warnings look like issues coming from the language server maybe?  But it seems like waiting on the work Damien mentioned above makes sense at this point.

**Comment by JosephTLyons** [comment]
Hey @rsese, followed your instructions exactly and still nothing on my end.  It seems lots of people are experiencing different issues.  I can't seem to force atom's new IDE features to work no matter what I do.  This will be a tough issue to troubleshoot for the developers for sure.
